README.md

```diff
@@ -68,6 +68,16 @@ Unlike [the base URLs](https://developers.google.com/photos/library/reference/re
 
 `imageUpdateDate` and `albumAddDate` are Unix epoch time.
 
+By default the url retrieve a small version of the image, to retrieve it to a specific dimension just append to the url `=wXXX-hXXX-no` where `XXX` are the dimensions
```

Comment:
What is the effect of `-no`? I cannot confilm why `-no` is required.

```bash
$curl -sSL https://lh3.googleusercontent.com/Pt3C6874cqkfeuIVL0XZ-UCsC6zLzeQmxq7T9-sDiPhyAgvJiKl_SCrvrMMkpuWuZ1TFkU65il
aZJrCbePRYo1q1qGTYvFV6J8gbYfZhhxQuXm2zXx6QDQkj0K-uBBUzozw7YLYQ5g=w640-h480-no | openssl sha256
(stdin)= be1c72df2d79f3a5bd59d077f66b5c2b2e7166307dc885ae914a30a9d11ff4bd

$curl -sSL https://lh3.googleusercontent.com/Pt3C6874cqkfeuIVL0XZ-UCsC6zLzeQmxq7T9-sDiPhyAgvJiKl_SCrvrMMkpuWuZ1TFkU65il
aZJrCbePRYo1q1qGTYvFV6J8gbYfZhhxQuXm2zXx6QDQkj0K-uBBUzozw7YLYQ5g=w640-h480 | openssl sha256
(stdin)= be1c72df2d79f3a5bd59d077f66b5c2b2e7166307dc885ae914a30a9d11ff4bd
```

Reply:
It might not be required if you say so, just found the url and tried it as is, as it was working I did not question the -no :)